packages/astro/src/core/build/index.ts

```diff
@@ -174,7 +174,8 @@ class AstroBuilder {
 		await runHookBuildStart({ config: this.settings.config, logging: this.logger });
 		this.validateConfig();
 
-		this.logger.info('build', `output: ${blue('"' + this.settings.buildOutput + '"')}`);
+		this.logger.info('build', `output: ${blue('"' + this.settings.config.output + '"')}`);
+		this.logger.info('build', `ssr: ${blue((this.settings.buildOutput == 'server' ? 'enabled' : 'disabled'))}`);
```

Comment:
```suggestion
		this.logger.info('build', `mode: ${blue((this.settings.buildOutput))}`);
```

We can call it build mode, 'cause that's how we call it internally. What do you think?

Reply:
yeah that sounds good